charts/sonarqube/templates/sonarqube-sts.yaml

```diff
@@ -64,6 +64,10 @@ spec:
         - name: "wait-for-db"
           image: {{ default "busybox:1.32" .Values.initContainers.image }}
           imagePullPolicy: {{ .Values.image.pullPolicy  }}
+          {{- if $securityContext := .Values.initContainers.securityContext }}
```

Comment:
reusing the `initContainers` scope is misleading

Reply:
Hi! Isn't this a initContainer? Should I create a value like wait-for-db.securityContext ?

Reply:
ah yeah you are right. sorry too many topics currently